PhysicsTools/NanoAOD/python/electrons_cff.py

```diff
@@ -144,61 +131,64 @@ def _get_bitmapVIDForEle_docstring(modules,WorkingPoints):
     EAFile_MiniIso = cms.FileInPath("RecoEgamma/ElectronIdentification/data/Fall17/effAreaElectrons_cone03_pfNeuHadronsAndPhotons_94X.txt"),
     EAFile_PFIso = cms.FileInPath("RecoEgamma/ElectronIdentification/data/Fall17/effAreaElectrons_cone03_pfNeuHadronsAndPhotons_94X.txt"),
```

Comment:
@jainshilpi 
are these miniIso and PFIso the most up to date ? they are labelled as 94X.

Reply:
@mariadalfonso this is the right file. For full run 2, our recommendation is to use the Fall94X IDs. So "RecoEgamma/ElectronIdentification/data/Fall17/effAreaElectrons_cone03_pfNeuHadronsAndPhotons_94X.txt" is fine.